packages/aws-cdk/lib/cli/user-configuration.ts

```diff
@@ -35,7 +35,7 @@ export enum Command {
   DOCTOR = 'doctor',
 }
 
-const BUNDLING_COMMANDS = [
+export const BUNDLING_COMMANDS = [
```

Comment:
I like the idea but i'm not a fan of exporting just for the sake of a unit test. It makes the unit test somewhat brittle because they rely heavily on implementation, and not on behavior. A better option would be to enforce each command to specify whether it is bundling or not, during command creation. So for example:

```ts
export interface Command {
  bundling: boolean;
  name: string;
}

export const Commands: Command[] = [
  { name: 'ls', bundling: false },
  ....
]
``` 

This also opens the door for additional command specific configuration we might need in the future. 
I recognize this would be a larger change, and maybe its worth syncing with @mrgrain as part of toolkit refactoring. Let me know what you think.

Reply:
+1 to @iliapolo suggestion. 

This will have to work very differently for the Toolkit, because `bundling` is something that controls `CloudAssembly` behavior. In the CLI we have a strict 1:1 mapping between `CloudAssembly` and CLI command. However in the toolkit, one can use the same `CloudAssembly` for different Toolkit actions. Not quite sure how to go about this yet. 🤔